actionpack/test/dispatch/system_testing/driver_test.rb

```diff
@@ -133,4 +133,17 @@ class DriverTest < ActiveSupport::TestCase
   ensure
     ::Selenium::WebDriver::Chrome::Service.driver_path = original_driver_path
   end
+
+  test "does not preload if :rack_test is set" do
+    called = false
```

Comment:
We can use a helper method to check that method calls or not. Like the following:

```ruby
  test "does not preload if :rack_test is set" do
    assert_not_called_on_instance_of(ActionDispatch::SystemTesting::Browser, :preload) do
      ActionDispatch::SystemTesting::Driver.new(:rack_test, using: :chrome)
    end
  end
```

Reply:
I updated the code to use the `assert_not_called_on_instance_of` method and  force-pushed it.